### PR TITLE
Improve the setup docs for Boskos

### PR DESCRIPTION
**Pull request by liztio**

### CODE REVIEW

File: boskos/deployment.yaml [hunk 1]
```diff
@@ -11,7 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-pods
+---
 apiVersion: v1
```
Comment (sebastienvas) [inline boskos/deployment.yaml]: I don't think the persistent volume is required anymore. @krzyzacy

Comment (krzyzacy) [reply boskos/deployment.yaml]: hummm I thought we still fall back to in-memory? (I haven't tried to get rid of it, your call)

File: boskos/README.md [hunk 2]
```diff
@@ -8,24 +8,26 @@ boskos is a resource manager service, that handles and manages different kind of
...
+    names:
+    - "account1"
+    - "account2"
 ```
 
 Type can be GCPProject, cluster, or even a dota2 server, anything that you
 want to be a group of resources. Name is a unique identifier of the resource.
 State is a string that tells the current status of the resource.
 
+The resources aren't much use without customization.
```
[the remark this answers is not in the document]
Comment (liztio) [reply boskos/README.md]: `use` is correct, but that whole line was errant, so I deleted it :)

**Pull request merged**